ex14v2_ch4.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+#step1: state constant
+WI_TAX_PERCENT = 5.5
+
+#step2: get inputs from user on order amount and state
+order_amount = float(input("Enter the order amount:$"))
+state = input("What state do you live in? Enter the state abbreviation (eg, MA): ")
+
+#step3: set up first conditional to see if user lives in WI
+if state == "WI":
+    print(f"You're a Wisonsin resident, and will be charged a tax of {WI_TAX_PERCENT}%.")
+    tax_amount = WI_TAX_PERCENT / 100 * order_amount
+    total_with_tax = tax_amount + order_amount
+    print("Your order amount before tax is ${:.2f}".format(order_amount)+", but with tax your grand total is ${:.2f}.".format(total_with_tax))
+#step4: set up conditional for users who don't live in WI
+else: 
```

Comment:
Perfect use of `else` that applies for everyone _but_ WI residents. This demonstrates a single fork going to two separate branches, which is modeled with an `if-else`.